style.css

```diff
@@ -0,0 +1,877 @@
+    top: 2rem;
+    margin: 2.5rem 1rem;
+  }
+
+  .nav-menu-link {
+    color: var(--tetiary-color);
+    letter-spacing: 1px;
+    font-size: 1.5rem;
+    font-weight: 600;
+  }
+
+  .red {
+    color: var(--secondary-color);
+  }
+
+  .nav-menu-btn.slide .bar:nth-of-type(1) {
+    transform: rotate(45deg) translateX(8.2px) translateY(6.2px);
+    background-color: var(--tetiary-color);
+  }
+
+  .nav-menu-btn.slide .bar:nth-of-type(2) {
+    opacity: 0;
+  }
+
+  .nav-menu-btn.slide .bar:nth-of-type(3) {
+    transform: rotate(-45deg) translateX(7.2px) translateY(-6.2px);
+    background-color: var(--tetiary-color);
+  }
+}
+
+@media screen and (min-width: 766px) {
+  .header-section {
+    background-size: cover;
+    background-repeat: no-repeat;
+  }
+
+  .nav-bar {
+    background-color: white;
+  }
+
+  .nav-menu-btn {
+    display: none;
+  }
+
+  .dblack-nav {
+    display: block;
+    padding-right: 7rem;
+  }
+
+  .nav-menu {
+    padding-right: 0.6rem;
+  }
+
+  .nav-menu-content {
+    align-items: center;
+  }
+
+  .nav-menu-item {
+    margin: 0 1rem;
+  }
+
+  .logo {
+    padding-left: 0.6rem;
+  }
+}
+
+@media screen and (min-width: 1000px) {
+  .dblack-nav {
+    padding-right: 14rem;
+  }
+
+  .nav-menu {
+    padding-right: 6rem;
+  }
+
+  .logo {
+    padding-left: 6rem;
+  }
+
+  .header-content {
+    margin-left: 5.5rem;
+  }
+}
+
+/* $-Header-Content-$ */
+.header-content {
+  z-index: 500;
+}
+
+h1 {
+  color: var(--secondary-color);
+  font-size: 1.5em;
+  font-weight: 900;
+  letter-spacing: 1px;
+  background: transparent;
+}
+
+.hello {
+  display: block;
+  font-size: 0.7em;
+  font-weight: 400;
+}
+
+.abb-info {
+  background-image: url('asset/bkgrnd.png');
+  background-size: cover;
+  border: 2px solid white;
+  width: 100%;
+  max-width: 28em;
+}
+
+.date {
+  display: block;
+  font-weight: 900;
+  font-size: 1.4em;
+  line-height: 2.4rem;
+}
+
+@media screen and (min-width: 766px) {
+  .header-content {
+    height: 65vh;
+    padding-left: 0.5em;
+  }
+
+  h1 {
+    font-size: 2.5rem;
+    font-weight: 900;
+  }
+
+  .abb-info {
+    background-image: url('asset/about-dektop.png');
+    width: 26rem;
+  }
+
+  .about-info {
+    max-width: 40rem;
+    margin: 0 auto;
+  }
+
+  .date {
+    font-size: 1.8rem;
+  }
+}
+
+/* $-Program-$ */
+.program {
+  flex-direction: column;
+  padding: 1rem;
+  z-index: -100;
+  opacity: 1;
+  background: radial-gradient(circle, transparent 20%, #1a1a1e 20%, #1a1a1e 80%, transparent 80%, transparent), radial-gradient(circle, transparent 20%, #1a1a1e 20%, #1a1a1e 80%, transparent 80%, transparent) 17.5px 17.5px, linear-gradient(#000 1.4000000000000001px, transparent 1.4000000000000001px) 0 -0.7000000000000001px, linear-gradient(90deg, #000 1.4000000000000001px, #1a1a1e 1.4000000000000001px) -0.7000000000000001px 0;
+  background-size: 35px 35px, 35px 35px, 17.5px 17.5px, 17.5px 17.5px;
+  background-color: #1a1a1e;
+  color: var(--tetiary-color);
+}
+
```

Comment:
- [ ] [REQUIRED] Please put a `hover effect` on the program `cards` like [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) and pay attention to the `height` and `width`!

![image](https://user-images.githubusercontent.com/63915024/190242062-bb0bc590-13cd-4d06-b717-f3cc917a9cd6.png)

Reply:
Interactions

Links
The home page should have a link in the menu to the about page.
The logo in the header links to the home page.
Mobile menu
When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
There are no guidelines for the mobile menu in the docs, but you should implement it so it is consistent with the design (colors, typography, spacings, etc.).
The mobile menu should have a close (X) button that closes the menu.



You should follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including:

Colors.
Typography: font face, size and weight (we suggest using [Lato](https://www.latofonts.com/) as it is a free font similar to the one used in the original design).
Layout: composition and space between elements.
The pages should look almost identical to the original design. Small adjustments like text or image changes are acceptable.

Reply:
Almost identical not exactly, please consider this

Reply:
- [ ] [REQUIRED] You must fulfill the [requirement before](https://github.com/UsirusenIkon/Allblack-Basketball-Camp-First-Capstone-/pull/1/files#r971204693) me because the content of this module is `"HTML&CSS"` which means you must fully respect the [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) provided by `Microverse`.